app/components/autocomplete/autocomplete.js

```diff
@@ -70,19 +99,18 @@ export default class Autocomplete extends PureComponent {
         if (atMentionCount + channelMentionCount + emojiCount + commandCount > 0) {
             containerStyle.push(style.borders);
         }
-
         return (
             <View>
                 <View style={containerStyle}>
                     <AtMention
                         cursorPosition={this.state.cursorPosition}
                         onResultCountChange={this.handleAtMentionCountChange}
-                        {...this.props}
+                        {...{...this.props, listHeight: this.listHeight()}}
```

Comment:
can you just add the prop as `listHeight={this.listHeight()}` ?

Reply:
I thought it wouldn't override... I can check again though.

Reply:
Oh then you can add it after the `{...this.props}` or make it so you pass `{...otherProps}` is much cleaner